hedera-cryptography/hedera-cryptography-tss/src/main/java/com/hedera/cryptography/pairings/api/Curve.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.cryptography.pairings.api;
```

Comment:
@nathanklick they are temporally located there.
Those classes WILL live in another project. That is why I exposed them that way.
This is done on purpose so that we don't have to refactor whoever decides to try this code.

Reply:
I spoke to @nathanklick regarding this and explained the intention.